src/app/components/feed/SaveFeed.js

```diff
@@ -621,27 +619,45 @@ const SaveFeed = (props) => {
             feed.id ? (
               <FormattedMessage
                 id="saveFeed.confirmationDialogTitle"
-                defaultMessage="Are you sure you want to update this shared feed?"
+                defaultMessage="Update Shared Feed?"
                 description="Confirmation dialog title when saving a feed."
               />
             ) : (
               <FormattedMessage
-                id="saveFeed.invitationConfirmationDialogTitle"
-                defaultMessage="Collaboration invitations"
-                description="Confirmation dialog title for feed collaboration invitations."
+                id="saveFeed.confirmationDialogTitleCreate"
+                defaultMessage="Create Shared Feed?"
+                description="Confirmation dialog title for creating a feed."
               />
             )
           }
           body={
             <div>
-              { feed.id &&
-              <p>
-                <FormattedMessage
-                  id="saveFeed.confirmationDialogBody"
-                  defaultMessage="Are you sure you want to update this shared feed?"
-                  description="Confirmation dialog message when saving a feed."
-                />
-              </p>
+              { feed.id ?
+                <p>
+                  <FormattedHTMLMessage
+                    id="saveFeed.confirmationDialogBody"
+                    defaultMessage="This shared feed will be available to all users of <strong>{name}</strong> and collaborating workspaces."
```

Comment:
Sentence begins with "Updates to" in the design for the update case

Reply:
Thanks for catching that, updated!